### ISSUE TITLE
[508] Certificate of Eligibility | Fix heading levels on the Introduction page

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#42354]
## Value Statement

**_As a_** screenreader user
**_I want_** the headers on the page to increment sequentially
**_So that_** I can understand the structure of the page

## Background Context

While testing the [Certificate of Eligibility (COE) introduction page](https://staging.va.gov/housing-assistance/home-loans/request-coe-form-26-1880/introduction):
- The not logged in state passes accessibility (axe) testing
- The logged in state changes the page content and the process list (subway map) headers do not pass accessibility (axe) testing

    <details><summary>Axe heading level issueTitle</summary>

    
    <img width="1068" alt="axe dev tools panel showing heading levels should only increase by one message" src="https://user-images.githubusercontent.com/136959/171952080-d5dd31ae-6d00-4c0d-afda-64970ffb4b6e.png"></details>

## Tasks

- [x] Replace `h3` with `h2` and add h3 font size class
- [x] Re-run axe check
- [ ] Add tests, if appropriate

## Definition of done

- [x] All tasks complete
- [x] All tests passing

**Comment by rocio-desantiago** [comment]
Looks like @scottymeyers may have completed 6 days ago :D

**Comment by scottymeyers** [comment]
@rocio-desantiago nope, I did other work on that page. But not these items.

**Comment by Mottie** [comment]
Ok, so I did duplicate this issue...
- No in-progress form
- No COE to download
- No COE status
![heading level jump from 1 to 3](https://user-images.githubusercontent.com/136959/173877238-19ca5b65-1814-40b1-9fef-6a3e52c81d37.png)

**Comment by rocio-desantiago** [comment]
@scottymeyers ah interesting. The other day I saw it in order. I'll look again. to put it in order. Should be h1, h2, h3 order right?

**Comment by Mottie** [comment]
Actually, I think in this case, we need to maybe add an h2 (not sure what text to include) above the "Request a Certificate of Eligibility" action link... changing all the headers inside the `va-procress-list` won't be fun
